Assets/Countly/Plugins/CountlySDK/Services/AbstractBaseService.cs

```diff
@@ -103,6 +103,13 @@ protected string TrimValue(string fieldName, string v)
                 object v = item.Value;
 
                 if (k == null || k.Length == 0 || v == null) {
+                    if (k == null || k.Length == 0) {
```

Comment:
remove the previous if check and add 'continue' to these two you have introduced after the logs. 
Also the second log is not correct. it will not be truncated, this key/value pair will be skipped. Even if it was not skipped we can not know if it will be truncated (can be smaller than threshold)